pkg/controllers/machine_controller.go

```diff
@@ -173,6 +194,44 @@ func (r *MachineReconciler) Start(ctx context.Context) error {
 	return nil
 }
 
+func (r *MachineReconciler) processVolumeSizeChanges(parent context.Context, log logr.Logger) {
```

Comment:
Can we use `ctx` instead of `parent` here.

Reply:
It was before `ctx` but for @lukas016 it was confusing hence I changed it.

Reply:
before you had anonymous function in start function with another context with the same name. here you can keep ctx as @afritzler suggest because you don't have anymore problem with the same names.

Reply:
@lukas016 To be precise the name `ctx` is still overloaded as you can see here `wait.UntilWithContext(ctx, func(ctx context.Context) {` .

But anyhow before and now I prefer to name a context like `ctx`.

Reply:
@lukasfrank yes you have right, my fault. But in this case you have previous problem, what about innerCtx? Because it little looks as closure but it isn't true.